### DISCUSSION TITLE
Nitro sniper modes

### DISCUSSION THREAD

**Discussion by WSrujan** [marioparaschiv/nitro-sniper#125] [Q&A] [answered]
what is 3 Modes - main, alts, both
does nitro will always be claim on main token?
i know this dumb question sorry

**Comment by InimicalPart** [comment, marked as answer]
When you put the tokens in the "alts" section in the .env file, it allows searching for nitro/giveaways on these accounts as well. When the codes are gotten then it redeems it on the main. And that's probably why the sniper won't work without the main token. The options `main`, `alts` and `both` are there to select where it should search for the token. Like `main` will only search for tokens on the main account and redeem them. But `alts` will search on the alt accounts and redeem on the main (recommended as it lowers the chance of your main getting banned) and `both` which searches for codes on the main and the alt accounts.

P.S No question is stupid or dumb :)
